src/astro/databases/base.py

```diff
@@ -51,6 +52,14 @@ def sqlalchemy_engine(self) -> sqlalchemy.engine.base.Engine:
         """Return Sqlalchemy engine."""
         return self.hook.get_sqlalchemy_engine()
 
+    def get_sqlalchemy_engine(self, table: Table):
+        return self.hook.get_sqlalchemy_engine()
+
+    def get_connection(self, table) -> sqlalchemy.engine.base.Connection:
```

Comment:
Missing type-hint for table, it may be worth calling `get_connection_from_table`